### DISCUSSION TITLE
No logs or alerts on Master

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#1999] [Unsupported Versions] [answered]
After completing setup on a forward node I do not see any logs or alerts. Netstat shows a salt connection to the forward node. I see logs on the monitoring interface of the forward node.  Not sure what else to look for.

**Comment by dougburks** [comment, marked as answer]
On the forward node, is Zeek writing logs to `/nsm/zeek/logs/current/`?

Have you checked the logs in `/opt/so/log/` for any additional clues?

**Reply by gcsrellis** [reply]
Yes, zeek logs are being written.  I dont see any logs that stand out with problems, although I'm no expert! I did see a filebeat connection over 5044 also.

**Reply by dougburks** [reply]
OK, so it sounds like Filebeat should at least be trying to send logs to Logstash on the manager. Have you checked the Logstash logs on the manager in `/opt/so/log/logstash/` for any additional clues?

**Reply by gcsrellis** [reply]
Log says server started on 5044 and 5644 and logstash api started. I just looked on the forward node and there is no /opt/so/saltstack/local. Could this be the isse? I was trying to verify homenet and i dont see the sls file. The directory path ends at /saltstack

**Reply by dougburks** [reply]
Nodes pull all of their salt information dynamically from the manager, so they shouldn't really have anything in /opt/so/saltstack/.

In Hunt or Kibana, try changing your search window to search from 2 days ago to 2 days from today.

**Reply by gcsrellis** [reply]
I tried that initially. Tried all different searchs.

**Reply by dougburks** [reply]
What do you get if you run `sudo so-redis-count` on the manager?

**Reply by gcsrellis** [reply]
Integer 716190

**Reply by dougburks** [reply]
When you built your manager, did you configure it as MANAGER or MANAGERSEARCH?
![image](https://user-images.githubusercontent.com/1659467/99709606-6dfc8680-2a6d-11eb-9530-98c376932910.png)

If MANAGER, do you have a search node connected to your manager that should be pulling logs out of that redis queue?

**Reply by gcsrellis** [reply]
I built it as manager and the sensor as forward.  So if I understand what you are saying is, I should have chosen managersearch so the manager can search and return the elastic queries? Man I feel silly now, lol.  Learning all the changes from 16 is a learning curve.

**Reply by dougburks** [reply]
Yes, you can either build a managersearch or build a manager and then add separate search nodes. For more information, please see https://docs.securityonion.net/en/2.3/architecture.html.
